src/videopython/generation/stability/text_to_image.py

```diff
@@ -65,11 +54,9 @@ def get_image_from_prompt(
                     "Your request activated the API's safety filters and could not be processed."
                     "Please modify the prompt and try again."
                 )
-
             if artifact.type == generation.ARTIFACT_IMAGE:
                 img = Image.open(io.BytesIO(artifact.binary))
-                img.save(filename)
```

Comment:
I would consider adding parameter `save` instead of removing it completely.

Reply:
Agreed, pushed the changes 👍🏻